tests/src/test/resources/ignored-links

```diff
@@ -1,3 +1,8 @@
+https://issues.redhat.com/browse/KEYCLOAK-18994
+https://developers.facebook.com/docs/development/
+https://developers.facebook.com/ 
+https://developers.facebook.com/docs/graph-api 
+https://developers.facebook.com/docs/facebook-login/access-tokens/ 
```

Comment:
Why are you adding these links to ignore?

Reply:
I would suggest removing these changes in this PR, and rather open a separate PR to resolve any broken links.

Reply:
Okay, @stianst.  To be precise, the links are not broken.  I tested them in a built version of the document and they all work, but they are reported as invalid, so that is why I added them to ignored-links.

So do you want me to remove the addition to ignored-links and request approval of the PR with a note they are not broken, then issue a separate PR adding these URLs to ignored-links?

Reply:
Before merging we review the results anyways and make sure broken links are not caused by the PR

Ignoring the links is not the right fix, we need to figure out why valid links are reported as broken instead.